aspnetcore/security/anti-request-forgery.md

```diff
@@ -163,25 +161,24 @@ public async Task<IActionResult> RemoveLogin(RemoveLoginViewModel account)
 }
 ```
 
-The ``ValidateAntiForgeryToken`` attribute requires a token for requests to action methods it decorates, including `HTTP GET` requests. If you apply it broadly, you can override it with the ``IgnoreAntiforgeryToken`` attribute.
+The `ValidateAntiForgeryToken` attribute requires a token for requests to action methods it decorates, including `HTTP GET` requests. If you apply it broadly, you can override it with the `IgnoreAntiforgeryToken` attribute.
```

Comment:
Don't code fence HTTP GET

Reply:
Why on earth would you decorate GET anyway? We don't support putting tokens in URLs for GET requests.